public/app/features/alerting/unified/components/expressions/Expression.tsx

```diff
@@ -51,7 +51,7 @@ export const Expression: FC<ExpressionProps> = ({
   onSetCondition,
   onUpdateRefId,
   onRemoveExpression,
-  onUpdateExpressionType,
+  onUpdateExpressionType, // this method is not used? maybe we should remove it
```

Comment:
(There's a few of these throughout alerting code, I have a WIP branch from a little while ago where I was removing them, will open a PR)

Reply:
(PR is here https://github.com/grafana/grafana/pull/93271)